src/main/java/org/cyclonedx/util/serializer/SerializerUtils.java

```diff
@@ -42,5 +43,23 @@ public static boolean shouldSerializeField(Object obj, Version version, String f
     }
   }
 
+  public static void serializeProperty(String propertyName,  Property prop, ToXmlGenerator xmlGenerator) throws IOException {
+    xmlGenerator.writeFieldName("environmentVar");
```

Comment:
Should this be `propertyName` instead?

Reply:
> Should this be `propertyName` instead?

yes, good catch , addressed here 3781b6471ae424b3ebee376a28ba5d7bb49cef5b